eng/versioning/update_versions.py

```diff
@@ -148,26 +148,6 @@ def update_versions_all(update_type, build_type, target_file, skip_readme):
                 if (file_name.endswith('.md') and not skip_readme) or (file_name.startswith('pom.') and file_name.endswith('.xml')):
                     update_versions(version_map, file_path)
 
-    # This is a temporary stop gap to deal with versions hard coded in java files.
```

Comment:
We can't remove this yet, there are still data files with hard coded versions. eng\versioning\version_data_java_files.txt. My recommendation is to add a file exists check on the file and if the file doesn't exist then skip the replacement.